crm_lead_vat/models/crm_lead.py

```diff
@@ -17,11 +22,7 @@ def _create_customer(self):
         """Add VAT to partner."""
         return super(Lead, self.with_context(default_vat=self.vat))._create_customer()
 
-    def _prepare_values_from_partner(self, partner):
+    def _prepare_address_values_from_partner(self, partner):
         """Recover VAT from partner if available."""
-        result = super(Lead, self)._prepare_values_from_partner(partner)
-        if not partner:
-            return result
-        if partner.vat:
-            result["vat"] = partner.vat
+        result = super(Lead, self)._prepare_address_values_from_partner(partner)
```

Comment:
Following from the above comment, you should undo this diff and leave it as it was before.

This way, we only alter the behavior when a database has the module installed, and not only when it is available.

Reply:
changes are done.